modules/virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume_{context}"]
+
+= Reclaiming persistent volumes for binding to a new persistent volume
+
+Use this procedure to reclaim a statically-provisioned persistent volume for reuse.
+
+.Prerequisite
+* Note the connection details of the persistent volume you want to reclaim.
+Use the Web Console or command line to display storage information for the
+virtual machine.
+
+.Procedure
+. Display the Storage Class using the Web Console or command line.
```

Comment:
This is not enough information. Present explicit steps for at least one one of these methods.

Reply:
. Enter the following command to display connection details for the persistent volume
you want to reclaim. Save the output for future reference.
+
----
$ oc describe pv <pv_name>
----

. Edit the `storageclass.yaml` file. Ensure that the Storage Class `reclaimPolicy` is set to `delete`.

Reply:
@bgaydosrh, the virt docs say that you can't modify this object's parameters: https://github.com/openshift/openshift-docs/blame/master/modules/virt-creating-storage-class.adoc#L12

It also seems like you'd need to take a look at the object that you made from that yaml file, not the yaml itself.